sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/ServiceVersion.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.common;
+
+/**
+ * The versions of Azure Storage supported by this client library.
+ */
+public enum ServiceVersion {
```

Comment:
Is it possible to have different service versions for blobs, files and queues?

Reply:
I will probably update this for Storage to have ServiceVersion's in each of the clients, similar to KeyVault.